src/core/store.ts

```diff
@@ -48,10 +93,31 @@ type StateListener = () => void
 type MountedAtoms = Set<AnyAtom>
 
 // store methods
+/**
+ * Read an atom's hidden [AtomState]. Derived atom state may be recomputed if they
```

Comment:
what's "hidden" mean?

Reply:
I will change it to "internal". I mean, this state is not part of the public API, and our consumer does not need to consider it.

Reply:
gotcha.